core-plugins/src/main/java/io/cdap/plugin/transform/LogParserTransform.java

```diff
@@ -216,12 +223,15 @@ private String getLog(StructuredRecord input) {
     }
   }
 
-  private void validateInputSchemaType(Schema.Type inputSchemaType) {
+  private void validateInputSchemaType(Schema.Type inputSchemaType, FailureCollector collector) {
     if (!Schema.Type.STRING.equals(inputSchemaType) && !Schema.Type.BYTES.equals(inputSchemaType)) {
-      throw new IllegalArgumentException(String.format(
-        "Unsupported inputType in schema, only Schema.Type.BYTES and Schema.Type.STRING are supported " +
-          "InputType: %s", inputSchemaType.toString()));
-
+      collector.addFailure(
+          String.format(
+              "Unsupported inputType '%s' in schema, only Schema.Type.BYTES and Schema.Type.STRING are supported",
+              inputSchemaType.toString()),
```

Comment:
use displayname

Reply:
https://github.com/data-integrations/google-cloud/blob/release/0.13/src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java#L357
something like this. This method is added to keep the schema types consistent in the error messages

Reply:
Fixed